tools/installer/_build.sh

```diff
@@ -22,12 +40,16 @@ for d in _copy _inst _build _build_osx $DIST; do
 done
 
 pushd "$EXAILE_DIR"
-git archive HEAD --prefix=_copy/ | tar -x -C tools/installer/
+build_git archive HEAD --prefix=_copy/ | build_tar -x -C tools/installer/
 popd
 
 pushd "$COPYDIR"
-make
-PREFIX=/usr DESTDIR="$DESTDIR" make install
+# Our Makefile relies on $PYTHON2_CMD to run Python commands
+export PYTHON2_CMD="${BUILD_ROOT}"/"${MINGW}"/bin/"${PYTHON_ID}".exe
+# help2man relies on perl-Locale-gettext, which lives in this directory
+export PERL5LIB="${BUILD_ROOT}"/usr/lib/perl5/vendor_perl
```

Comment:
If we change the build command to be `build_make compile locale` then we wouldn't need to worry about the manpage nonsense.